build/cloneReplace.js

```diff
@@ -0,0 +1,581 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
This file was created by running the plugin in dev mode. I would recommend gitignoring this file so that we are only ever committing the `bundle.min.js` and `bundle.min.js.map` files.